resources/js/app/components/object-types-picker/object-types-picker.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <div class="object-types-picker">
+        <Treeselect
+            :placeholder="msgObjectTypes"
+            :options="types"
+            :async="false"
+            :clearable="false"
+            :disable-branch-nodes="true"
+            :multiple="true"
+            value-format="object"
+            v-model="selectedTypes"
+            @input="changeTypes"
+        />
+    </div>
+</template>
+<script>
+import { Treeselect } from '@riophae/vue-treeselect'
+import { t } from 'ttag';
+import '@riophae/vue-treeselect/dist/vue-treeselect.css'
+
+export default {
+    name: 'ObjectTypesPicker',
+
+    components: {
+        Treeselect,
+    },
+
+    props: {
+        initialSelected: {
+            type: Array,
+            default: () => [],
```

Comment:
Why not `default: []`?

Reply:
It's a little verbose, but better in ES6